### PR TITLE
Feature/ele 660 add sortable label in revisorschema

### PR DESCRIPTION
**Pull request by hugowetterberg**
* revamp keyword handling, add sortable

This required a bit of a change in the equality/superset calculations as
comparison now has to take sub-fields into account.

Adds sortable field support with ICU icu_collation_keyword subfields.

* feature flag index features for smooth upgrades

### CODE REVIEW

File: index/index.go [hunk 1]
```diff
@@ -474,6 +474,13 @@ func (idx *Indexer) ensureIndex(
 		fmt.Sprintf("%s-%s", indexTypeRoot, config.Language),
 	}
 
+	// Add a custom normaliser for sort fields.
+	config.Settings.Settings.Analysis.SetNormalizer(
+		"lowercase_trim", OpensearchNormaliser{
+			Type:   "custom",
+			Filter: []string{"lowercase", "trim"},
```
Comment (fred-o) [inline index/index.go:481]: For true multilingual support I wonder if this is enough. We should problable look into using something like `icu_collation_keyword` (https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu-collation-keyword-field.html) and use the language settings for the index.

Comment (fred-o) [reply index/index.go:481]: This would replace the `type` of the sort field and ditch the need for a custom normalizer.

**Review by fred-o — COMMENTED** [review]
For the most part everything looks good to me, but see below for discussion about language-specific collation.

File: index/index.go (hunk 1 above)
Comment (hugowetterberg) [reply index/index.go:481]: Nice! I wonder how that works when sorting across multiple indexes with different collations.

Comment (hugowetterberg) [reply index/index.go]: Sorting works fine across indexes with different collations. It doesn't return a 400/error and the sort representation seems to be close enough to not result in really strange ordering.

**Review by fred-o — APPROVED** [review]

**Pull request merged**